### PR TITLE
chore(CONTRIBUTING): explain contribution process

### PR DESCRIPTION
**Pull request by technosophos**
Issue #265 suggests that we should have contribution guidelines that are
more specific to the Helm project. This adds a few high-level notes and
then points the user back to the Deis guidelines.

### CODE REVIEW

**Comment by technosophos** [comment]
Would love to get a review from @bacongobbler and @mboersma -- and some suggestions as to whether any of this should make its way to the prototype repo.

**Comment by bacongobbler** [comment]
I'd say everything but the `go lint/fmt/vet` comment could be moved upstream. A lot of it is process documentation which is nice but some projects do not require Go, for example deis/postgres :)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,5 +1,38 @@
 # How to Contribute
 
-This project is part of Deis. You can find the latest contribution
-guidelines [at the Deis project](https://github.com/deis/deis/blob/master/CONTRIBUTING.md).
+We love getting Pull Requests (PRs). To make sure we keep code quality
+and consistency high, we do have some process in place. In a nutshell:
+
+- Code should follow Go coding standards and pass `go lint` and `go vet`
```
Comment (sgoings) [inline CONTRIBUTING.md]: We should also say something about these tools being run automatically on pull requests (which we should set up for those projects that don't have PR builds enabled with anything valuable).

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,5 +1,38 @@
...
+- Awaiting review: The PR is ready to be considered for merging. If a PR
+  does not have this, we assume that it's a work in progress (even if
+  the issue does not say WIP)
+- Proposal: something we're discussing, but haven't decided on
+- Enhancement: a feature or chore
+- Bug: a bug
+- Needs manual testing: one of the core team must manually test before
+  LGTM
+- LGTM1: First LGTM (Looks Good To Me)
+- LGTM2: Second LGTM; this means PR can be merged
```
Comment (sgoings) [inline CONTRIBUTING.md]: Don't we have to merge the PR for them though? maybe: "this means the PR will soon be merged by a project maintainer"?

**Pull request merged**